src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -40,6 +40,9 @@ public static InquiryVo getCreateInquiryResponseData(Inquiry inquiry, InquiryVo
         inquiryVo.setInquiryDate(LocalizationUtil.getFormattedDate(inquiry.getInquiryDate()));
         inquiryVo.setGender(Gender.enumToString(inquiry.getGender()));
         inquiryVo.setComputerKnowledge(ComputerKnowledge.enumToString(inquiry.getComputerKnowledge()));
+        inquiryVo.setClosingStatus(FollowUpStatus.enumToString(inquiry.getClosingStatus()));
```

Comment:
This will be null in most of the update requests for the inquiry and will be only filled while closing the inquiry. Make sure null check is done in enum